builtin/mainmenu/settings/dlg_file_browser.lua

```diff
@@ -0,0 +1,156 @@
+-- Minetest
+-- Copyright (C) 2020 Jean-Patrick Guerrero <jeanpatrick.guerrero@gmail.com>
+-- Copyright (C) 2020 EvidenceBKidscode
+-- Copyright (C) 2023 ROllerozxa
+--
+-- This program is free software; you can redistribute it and/or modify
+-- it under the terms of the GNU Lesser General Public License as published by
+-- the Free Software Foundation; either version 2.1 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU Lesser General Public License for more details.
+--
+-- You should have received a copy of the GNU Lesser General Public License along
+-- with this program; if not, write to the Free Software Foundation, Inc.,
+-- 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+local PATH = os.getenv("HOME") -- Linux & similar
+		or os.getenv("HOMEDRIVE") .. os.getenv("HOMEPATH") -- Windows
```

Comment:
probably have to do this to fix the issue @grorp mentioned
```suggestion
		or (os.getenv("HOMEDRIVE") .. os.getenv("HOMEPATH")) -- Windows
```